test/integration/attachment.test.js

```diff
@@ -99,17 +99,17 @@ describeAuthOnly('attachments - authenticated', function () {
   it('should return from the octet route', async function () {
     const attachment = '00000000-0000-1000-8000-000000000001'
     const response = await getAttachmentRouteOctet(attachment, app, authToken)
-    expect(response.status).to.equal(200)
+    expect(response.status).to.equal(201)
   })
 
-  it('should return 406 when requesting json from the octet route', async function () {
-    const attachment = '00000000-0000-1000-8000-000000000001'
-    const response = await getAttachmentRouteJSON(attachment, app, authToken)
-    expect(response.status).to.equal(406)
+  it('should return 500 when requesting json from the octet route', async function () {
+    const attachment = '00000000-0000-1000-9000-000000000001'
+    const response = await getAttachmentRouteOctet(attachment, app, authToken)
+    expect(response.status).to.equal(500)
   })
 
   it('should return 404 when requesting incorrect ID', async function () {
-    const attachment = '00000000-0000-1000-8000-000000000002'
+    const attachment = '00000000-0000-1000-8000-000000040002'
```

Comment:
why was this updated?

Reply:
? Must have happened during a pairing session